xmtp_mls/src/groups/device_sync/backup.rs

```diff
@@ -156,115 +132,111 @@ mod tests {
         assert_eq!(messages.len(), 1);
     }
 
-    #[tokio::test]
+    #[xmtp_common::test(unwrap_try = "true")]
```

Comment:
wasn't sure how i felt about this, since i'm a bit of a purist and transforming a syntax/language feature in-place just feels wrong, but it being opt-in makes alot of sense IMO and i'm definitely for it if it increases test readability